### ISSUE TITLE
Keras Model Import Failing Out of the Box

### ISSUE THREAD

**Issue by mobiusinversion** [deeplearning4j/deeplearning4j#4523]
#### Issue Description

I saved an h5 file using Keras, then I tried to import it into DL4J.  It fails with an InvalidKerasConfigurationException.

     org.deeplearning4j.nn.modelimport.keras.InvalidKerasConfigurationException: 
         Could not determine number of outputs for layer: no output_dim or nb_filter field found. 
         For more information, see http://deeplearning4j.org/model-import-keras.

From python:

     model = Sequential()
     model.add(Dense(10, activation='sigmoid', input_dim=5))
     model.add(Dense(3, activation='sigmoid'))
     model.compile(optimizer='rmsprop',
              loss='binary_crossentropy',
              metrics=['accuracy'])
    ...
    model.fit(data, labels, epochs=200000, batch_size=5)
    model.save('my_model.h5')
    
Then in Java 

    public static MultiLayerNetwork importKerasModel(String hdf5FileName) throws Exception {
        try {
            return KerasModelImport.importKerasSequentialModelAndWeights(hdf5FileName);
        } catch (Exception e) {
            logger.error("Could not load Keras model", e);
            throw e;
        }        
    }

Why is this failing and how do I fix it?

#### Version Information

Please indicate relevant versions, including, if relevant:

* Deeplearning4j version: 0.9.1
* platform information: Mac OSX, Java 1.8, Python 2.7.10, Keras 2.1.2

**Comment by agibsonccc** [comment]
Use https://deeplearning4j.org/snapshots 0.9.1 only supports keras 1.x.
